addons/fleet/models/fleet_vehicle_model.py

```diff
@@ -4,6 +4,19 @@
 from odoo import _, api, fields, models
 
 
+FUEL_TYPES = [
+        ('gasoline', 'Gasoline'),
```

Comment:
Could be de-indented

Reply:
Yes sure no need to take that much space